src/Middleware/RequestThrottling/src/SemaphoreWrapper.cs

```diff
@@ -1,12 +1,10 @@
 using System;
-using System.Collections.Generic;
-using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.AspNetCore.RequestThrottling
 {
-    internal class SemaphoreWrapper : IDisposable
+    public class SemaphoreWrapper : IDisposable
```

Comment:
Is this meant to be public? Do we want this API surface?

Reply:
good catch